app/views/groups/samples/index.turbo_stream.erb

```diff
@@ -11,6 +11,7 @@
                       pagy: @pagy
                     } %>
 
-<%= turbo_stream.update "group_samples_hidden_search_value" do %>
+<%= turbo_stream.update "group_samples_hidden_values" do %>
   <%= render Ransack::HiddenSortFieldComponent.new(@q) %>
+  <%= hidden_field_tag "metadata", params[:metadata] %>
```

Comment:
**nitpick (blocking):** are you sure that this is needed, since you moved the `metadata` param into the `search` form already.

Reply:
Nope, it is not, thanks.  Removed in 2e86100b703a4c8e4f3e65c424d0cf393bf1d3d4